searchcore/src/vespa/searchcore/config/ranking-expressions.def

```diff
@@ -0,0 +1,5 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=vespa.config.search.core
+
```

Comment:
Is it so tightly coupled with rank-profiles config that it should be added to rank-profiles.def  ?
Or is it better to keep them as separate configs.

Reply:
I did it the same way as constants and onnx models were done. Putting it into rank-profiles.def would make it a separate top-level array that needs to be handled separately form the rest of the config. We do need some extra wiring to add a new config though. Maybe we should combine config about distributed files somehow... @geirst any thoughts?

Reply:
I suggest we add this one now, and then we consider consolidation afterwards.